
    public InternalError(Throwable cause) {
        super(cause);
    }

}
